src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -12,8 +12,7 @@ info:
     url: https://github.com/slovensko-digital/autogram/blob/main/LICENSE
   version: 1.99.9
 servers:
-  - url: http://localhost:37200
-  - url: https://localhost:37200
+  - url: /
```

Comment:
Toto nebude fungovat dobre ak otvoris dokumentaciu tadeto - https://generator3.swagger.io/index.html?url=https://raw.githubusercontent.com/slovensko-digital/autogram/main/src/main/resources/digital/slovensko/autogram/server/server.yml (tak to mame v readme)

Reply:
upravene podla #220